src/main/grammars/RustParser.bnf

```diff
@@ -113,17 +113,32 @@ File ::= [ SHEBANG_LINE ] InnerAttr* Items
 // Attributes
 ///////////////////////////////////////////////////////////////////////////////////////////////////
 
-fake Attr ::= MetaItem
+fake Attr ::= MetaItem {
+  extends = "org.rust.lang.core.psi.ext.RsStubbedElementImpl<?>"
+  stubClass = "org.rust.lang.core.stubs.RsPlaceholderStub"
+}
```

Comment:
Yep, stubbing all attributes seems like the right approach to me! And looks like we can now remove `val langAttribute: String?` from `RsTraitItemStub`, and `val pathAttribute: String?` from `RsModDeclItemStub`.